### DISCUSSION TITLE
I am not able to join Slack

### DISCUSSION THREAD

**Discussion by Praashh** [json-schema-org/community#633] [Q&A]
Hey there! I am not able to join the slack, can anyone help to get into slack, getting this
![image](https://github.com/json-schema-org/community/assets/99237795/58eb7954-26c5-4536-827f-36f89db6842f)

**Comment by benjagm** [comment]
Thanks for letting us know! Can you please try again?

**Reply by msaadg** [reply]
Hey Benjamin, I'm also not able to join Slack. I'm using the link from https://github.com/json-schema-org/community/blob/main/programs/mentoring/gsoc/CONTRIBUTOR-GUIDANCE.md

<img width="1407" alt="Screenshot 2024-03-05 at 10 32 46 PM" src="https://github.com/json-schema-org/community/assets/110264527/fb49e94a-20a8-4ac4-b0d1-3ccb68f6a86b">

**Comment by benjagm** [comment]
please try now

**Reply by nexpectArpit** [reply]
hey benjagm, i am also facing the same issue, it says "this link is no longer active" on slack

**Comment by zapotocnylubos** [comment]
Hello, I just hit the same problem.
Link on the homepage does not work - result is the same as OP posted.

**Comment by katara-Jayprakash** [comment]
this link is still not working its say we are not registered with json.schema. how a new commer can join json schema slack channel?
